packages/react-composites/src/composites/CallWithChatComposite/EmbeddedChatPane.tsx

```diff
@@ -46,16 +65,23 @@ export const EmbeddedChatPane = (props: {
 
   if (props.mobileView) {
     return (
-      <MobilePane
+      <WithLocalAndRemotePIP
+        callAdapter={props.callAdapter}
         hidden={props.hidden}
-        dataUiId={'call-with-chat-composite-chat-pane'}
-        onClose={props.onClose}
-        activeTab="chat"
-        onChatButtonClicked={props.onChatButtonClick}
-        onPeopleButtonClicked={props.onPeopleButtonClick}
+        modalLayerHostId={props.modalLayerHostId}
+        styles={pipStyles}
       >
-        {chatComposite}
-      </MobilePane>
+        <MobilePane
```

Comment:
Implementation is throwing me a little here and it causes the position shift between chat and people panes. Can you check with AlexP if the placement of the PiPiP should remain unchanged when shifting between people and chat pane? it seems odd to have it suddenly move. I believe the implementation is also causing the PiPiP refresh between changing side panes.

I think we need to have the modal live outside of the specific chat and people panes, and inside a generic component such that is doesn't need to be recreated if the user switches between panes

Rather than having two seperate panes as we do now:
```
- CompositeContainer
  | - Chat Side Pane
  | - People Side Pane
```

I would suggest something more along the lines of:
```
- CompositeContainer
 | - GenericSidePane
    | - Side Pane Header
    | - Side Pane Contents (i.e. PeoplePaneContents / ChatPaneContents)
    | - PiPiP
```
In jsx an impl could be something like:
```
<SidePane props: { selectedHeader: 'chat' | 'people'; children } >
  <SidePaneHeader selectedHeader={props.selectedHeader />
  <SidePaneContents>{props.children}</SidePaneContents>
  <PiPiP />
</SidePane>
```
(also other ways to structure this, just a quick suggestion, the key idea being the seperation of sidepane contents and the PiPiP so they are siblings and changes to one doesn't cause a re-render of the other)

This means when switching between people pane and chat pane, only the pane contents should refresh -- the PiPiP should remain in place and with no render changes

Reply:
Talked with Alex P. and you are right there should only be one PiPiP. Simplified the pane logic similar to what you have proposed. Thanks.